generators/dotnetcore-server-interfaces/PrincipleStudios.OpenApiCodegen.Server.Mvc.Analyzers/templates/controller.handlebars

```diff
@@ -49,9 +49,9 @@ namespace {{packageName}}
             }}{{#if isHeaderParam}}global::Microsoft.AspNetCore.Mvc.FromHeader(Name = "{{rawName}}"){{/if}}{{!
             }}{{#if required}}, global::System.ComponentModel.DataAnnotations.Required{{/if}}{{!
             }}{{#if pattern}}, global::System.ComponentModel.DataAnnotations.RegularExpression(@"{{{escapeverbatimstring pattern}}}"){{/if}}{{!
-            }}{{#if minLength.HasValue}}, global::System.ComponentModel.DataAnnotations.MinLength({{minLength}}){{/if}}{{!
-            }}{{#if maxLength.HasValue}}, global::System.ComponentModel.DataAnnotations.MaxLength({{maxLength}}){{/if}}{{!
-            }}{{#if minimum.HasValue}}{{#if maximum.HasValue}}, global::System.ComponentModel.DataAnnotations.Range({{minimum}}, {{maximum}}){{/if}}{{/if}}{{!
+            }}{{#if HasMinLength}}, global::System.ComponentModel.DataAnnotations.MinLength({{minLength}}){{/if}}{{!
+            }}{{#if HasMaxLength}}, global::System.ComponentModel.DataAnnotations.MaxLength({{maxLength}}){{/if}}{{!
+            }}{{#if HasMinimum}}{{#if HasMaximum}}, global::System.ComponentModel.DataAnnotations.Range({{minimum}}, {{maximum}}){{/if}}{{/if}}{{!
```

Comment:
Another bug that I found - apparently `HasValue` wasn't accessible. I'm guessing because `decimal?` became `null | number` in the JS serialized for handlebars.